RHEL6_7/system/java-1.8.0-ibm/remove-java-1.8.0-ibm-pre-upgrade.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+VENDOR="$(rpm -q --queryformat "%{VENDOR}" java-1.8.0-ibm 2>/dev/null)"
+if [ $? -eq 0 ] && [ "$VENDOR" = "Red Hat, Inc." ]; then
```

Comment:
This is not necessary as you did the test earlier in the check script. But you can keep it, it is not wrong.